package.json

```diff
@@ -31,7 +31,7 @@
     "coap-cli": "0.4.0",
     "command-node": "0.1.1",
     "mongodb": "2.2.35",
-    "mongoose": "4.13.12",
```

Comment:
Line in CNR:

```
- Update MongoDB and Mongoose version to 2.2.35 and 4.13.12 respectively
```

should be adjusted.

Reply:
Fixed with 5eef71c1472478fae00da8de262b68b6173c34b5